index.js

```diff
@@ -27,6 +27,10 @@ alfy.fetch('https://api.npms.io/v2/search', {
 					},
 					cmd: {
 						subtitle: `${pkg.version} published at ${dateFormat('yyyy-dd-MM', new Date(pkg.date))} by ${(pkg.author && pkg.author.name) || pkg.publisher.username}`
+					},
+					ctrl: {
+						arg: pkg.name,
+						subtitle: `Copy Package ${pkg.name}`
```

Comment:
This makes it sound like it's going to copy the package itself.

Let's use:

> Copy package name

Reply:
This is incorrectly resolved. The casing is not what I wrote and the `${pkg.name}` should not be included.